experiments/integrated/ozark/ozark.jl

```diff
@@ -268,6 +268,20 @@ function diurnal_avg(series)
         [mean([daily_data[i][j] for i in 1:num_days]) for j in 1:daily_points]
     return daily_avgs
 end
+
+# This function scales 2 data series to the same length and matches their
+# indices such that they may be used to compute GOF stats
+function match_indices(model, data)
+    dpm = dt * n / DATA_DT
+    model_per_data = 1
+    if dpm < 1
+        model_per_data = Int64(1 / dpm)
+        dpm = 1
+    end
+    data_per_model = Int64(dpm)
+    return model[1:model_per_data:end], data[1:data_per_model:end]
+end 
+
```

Comment:
This works when dpm is an integer, but otherwise there is an error:
```
DATA_DT = 1800
dt = 45
n = 1
data= 0:DATA_DT:(24*3600-DATA_DT)
model = 0:n*dt:(24*3600-n*dt)

m, d = match_indices(model, data);
# looks good!
m == d


DATA_DT = 1800
dt = 400
n = 1
data= 0:DATA_DT:(24*3600-DATA_DT)
model = 0:n*dt:(24*3600-n*dt)
m, d = match_indices(model, data);
# errors
```

Im not sure how general we need to be. we could add an @assert statement that n*dt mod DATA_DT is zero or vice versa?

I think the other way to do this would be with linear interpolation - for example, if the data is sparser than the model output, we would use the model output nearest to the data at a given time in the data and interpolate the model output to that time in the data. We'd leave the data untouched. Linear interpolation itself is straightforward, but Im not sure how fast algorithm are at finding the bracketing times on either side.

anyways, just a thought. if you want to leave this as is and given an @assert plus some comments that's fine too.

Reply:
(we can also use Interpolations.jl directly if that sounds easiest, instead of implementing linear interpolation ourselves, if you want to go that route)

Reply:
Ah! You're right, I meant to be performing integer division to avoid this error which is what I did in my previous code.

I think you're also right that interpolation is a better solution. I was thinking of this PR of a quick and dirty way of fixing the bug so that the main branch code could work for sites with slower data (i.e Harvard Forest) and then including a better solution within the data tools we will need for the PR with the general experiment script.

Reply:
ok! sounds fine to me.